commands/build/lib/build.js

```diff
@@ -66,14 +67,22 @@ const config = ({
 
   return {
     input: {
+      onwarn(warning, warn) {
+        // Suppress circular dep. warnings to avoid noise (d3 circular dep. warnings...)
+        if (warning.code === 'CIRCULAR_DEPENDENCY') return;
+        warn(warning);
```

Comment:
There's a risk with supressing things. We might want to for specific reasons, but along comes someone else who needs to see it.
Could we do it as a --silent or --verbose mode? Best would be if we use the same as rollup and pass that flag along.

Reply:
Yeah, you are right. I read somewhere that rollup supports circular dependencies and therefore though it was OK to suppress these warning. But after some more reading and thinking, this only involves certain types of circular dependencies (of course) - hence, it's better to not suppress these warnings, as you said. - Done!